### PR TITLE
DEV-3518-feat(piscine-go): add solution/test for new go exercise fooddeliverytime

### PR DESCRIPTION
**Pull request by tiagocollot**
Created and added the solution and test for the new go exercise fooddeliverytime.

### CODE REVIEW

**Review by jotapero — APPROVED** [review]
Everything looks good

File: solutions/fooddeliverytime.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	}
+
+	if order == "burger" {
+		return burger.preptime
+	} else if order == "chips" {
+		return chips.preptime
+	} else if order == "nuggets" {
+		return nuggets.preptime
+	} else {
+		return 404
```
Comment (eslopfer) [inline solutions/fooddeliverytime.go:27]: There isn't any instruction in the description to return 404.

File: solutions/fooddeliverytime.go [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+		preptime: 10,
+	}
+
+	nuggets := food {
+		preptime: 12,
+	}
+
+	if order == "burger" {
+		return burger.preptime
+	} else if order == "chips" {
```
Comment (eslopfer) [inline solutions/fooddeliverytime.go:22]: Please consider if you really want to use else statements after return.

**Review by eslopfer — CHANGES_REQUESTED** [review]

File: solutions/fooddeliverytime.go (hunk 1 above)
Comment (tiagocollot) [reply solutions/fooddeliverytime.go:27]: Yes, it was missing the instruction in the exercise subject, it is fixed. Thank you for noticing that!

File: solutions/fooddeliverytime.go (hunk 2 above)
Comment (tiagocollot) [reply solutions/fooddeliverytime.go:22]: I have been writing  the else if statement after return just to not lose the logic process when developing the code.
I'll take your advice into consideration and will implement it in the new exercises to come.

**Review by eslopfer — APPROVED** [review]
LGTM! :rocket:

**Pull request merged**